webpack.dev.config.js

```diff
@@ -177,6 +177,11 @@ module.exports = (env = {}) => {
             '@gnosis.pm/pm-js',
           ],
         },
+        plugins: [
+          new webpack.EnvironmentPlugin({
+            NODE_ENV: 'development',
```

Comment:
Is this also needed in webpack.prod?

Reply:
@andre-meyer No, for some reason, it minifies the code by default. This is done to prevent minification